include/Foundation/NSKeyValueCoding.h

```diff
@@ -48,16 +48,16 @@ FOUNDATION_EXPORT NSString* const NSUnionOfSetsKeyValueOperator;
 - (id)valueForUndefinedKey:(NSString*)key;
 - (NSMutableArray*)mutableArrayValueForKey:(NSString*)key;
 - (NSMutableArray*)mutableArrayValueForKeyPath:(NSString*)keyPath;
-- (NSMutableSet*)mutableSetValueForKey:(NSString*)key STUB_METHOD;
-- (NSMutableSet*)mutableSetValueForKeyPath:(NSString*)keyPath STUB_METHOD;
-- (NSMutableOrderedSet*)mutableOrderedSetValueForKey:(NSString*)key STUB_METHOD;
-- (NSMutableOrderedSet*)mutableOrderedSetValueForKeyPath:(NSString*)keyPath STUB_METHOD;
+- (NSMutableSet*)mutableSetValueForKey:(NSString*)key;
```

Comment:
Why not? You've been known to do that before :)

---
In reply to: [113270799](https://github.com/Microsoft/WinObjC/pull/2610#discussion_r113270799) [](ancestors = 113270799)